src/Snackbar/Snackbar.react.js

```diff
@@ -68,16 +67,18 @@ function getStyles(props, context) {
             local.message,
             props.style.message,
         ],
-        actionContainer: [
-            snackbar.actionContainer,
-            local.actionContainer,
-            props.style.actionContainer,
-        ],
-        actionText: [
-            snackbar.actionText,
-            local.actionText,
-            props.style.actionText,
-        ],
+        button: {
+            container: [
+                snackbar.button.container,
+                local.button.container,
+                props.style.button.container,
+            ],
+            text: [
+                snackbar.button.text,
+                local.button.text,
+                props.style.button.text,
+            ],
+        },
```

Comment:
But it's not because of this, it's because you create button style in `getTheme.js`, isn't it? In this case, we should take default setting directly from `context.uiTheme.button` and merge it with, `snackBar.propTypes.button.style`. Should work, right?